features/build_runner/fastlane_build_runner.rb

```diff
@@ -51,8 +51,14 @@ def run(completion_block: nil, &block)
           #
           #   {"type":"success","message":"Driving the lane 'ios beta'","html":"<p class=\"success\">Driving the lane 'ios beta'</p>","time"=>...}
           #
-          row[:html] = FastlaneOutputToHtml.convert_row(row)
-          block.call(row)
+          # Also we use our custom BuildRunnerOutputRow class to represent the current row
+          current_row = FastlaneCI::BuildRunnerOutputRow.new(
```

Comment:
might be good to extract this into its own method

Reply:
👍